### ISSUE TITLE
Impossible to create more than one blog article with cyrillic title

### ISSUE THREAD

**Issue by kkovrizhenko** [aldryn/aldryn-newsblog#297]
If you create article with only cyrillic symbols for the first time, slog will be "untitled-article".
To add second one is impossible.

Please see traceback: http://dpaste.com/28TQSE2

**Comment by intellectronica** [comment]
Maybe we should really start using https://github.com/un33k/python-slugify - it handles cyrillic text beautifully.

**Comment by mkoistinen** [comment]
Agreed. We have requests for this elsewhere too. I'll likely add this to translation-tools soon, then all the addons will get that support.

**Comment by vizo** [comment]
:+1:
